app/gui/view/graph-editor/src/builtin/visualization/java_script/scatterPlot.js

```diff
@@ -806,6 +819,10 @@ class ScatterPlot extends Visualization {
             zoom.transformedScale.yScale.domain(domainY)
 
             self.zoomingHelper(zoom.transformedScale, boxWidth, scatter, points)
+
+            self.bounds = null
+            self.limit = 32
```

Comment:
is there anyway we can amend the limit from the UI?

Reply:
No, not right now. Easy to change the initial constant. To adjust the value from UI we need to come up with some gesture to do so.

Reply:
Spoke with Ned and Sylwia, I think if we plot 1000 points then its absolutely fine and won't need to amend the values.

Reply:
OK, set to `2^10` in [6dbd742](https://github.com/enso-org/enso/pull/3655/commits/6dbd74202f099e8e17491f430d56b55c22888f9e). Showing 1024 elements is more than enough to fill 25cm2. 

![1024 elements](https://user-images.githubusercontent.com/26887752/185550705-54f45a4a-94ea-47f0-bf18-bca3bb451949.png)

I had to use _Chrome Dev Tools_ to verify the `scatterPlot.js` works only with 1024 elements and not 20000.